ts/indicators/indicators.src.ts

```diff
@@ -62,6 +63,12 @@ declare global {
             yData: Array<(number|null)>;
         }
 
+        interface IndicatorMultipleNullableValuesObject {
+            values: Array<Array<(number|null)>>;
+            xData: Array<(number|null)>;
+            yData: Array<Array<(number|null)>>;
+        }
+
         interface IndicatorMultipleValuesObject {
             values: IndicatorValuesObject['values'];
             xData: IndicatorValuesObject['xData'];
```

Comment:
The typing is inconsistent between the interfaces. Please make sure that the `master` gets the fixed version of one of the other PRs and this does not overwrite it again.